src/components/Input/Input.test.js

```diff
@@ -60,6 +60,22 @@ describe('<Input />', () => {
     );
   });
 
+  it('should only render <label> if label prop is provided', () => {
+    props = {
+      valid: false,
+      message: 'Derp!',
+    };
+    wrapper = mount(withTheme(<Input {...props} />));
+
+    expect(wrapper.find('label').length).toBe(0);
+
+    wrapper = mount(
+      withTheme(<Input {...{ ...props, label: 'Woohoo, label time!' }} />)
+    );
+
+    expect(wrapper.find('label').length).toBe(1);
+  });
```

Comment:
Maybe also add an example with no label for comparison?

Reply:
Thats what the couple of lines above are doing, checks that without `label`, `length === 0`.